client/src/components/PetalModal.js

```diff
@@ -84,12 +84,12 @@ function PetalModal(props) {
   const [powersQuestion, setPowersQuestion] = useState("");
   const [challengesQuestion, setChallengesQuestion] = useState("");
   //state color for petals
-  const [peaksPetal, setPeaksPetal] = useState("yellow");
-  const [aspirationsPetal, setAspirationsPetal] = useState("yellow");
-  const [peoplePetal, setPeoplePetal] = useState("yellow");
-  const [principlesPetal, setPrinciplesPetal] = useState("yellow");
-  const [powersPetal, setPowersPetal] = useState("yellow");
-  const [challengesPetal, setChallengesPetal] = useState("yellow");
+  const [peaksPetal, setPeaksPetal] = useState("white");
```

Comment:
Looks great! One improvement to make would be to have a constant that is `DEFAULT_COLOR = "white"` which you can use everywhere instead of hardcoding "white".